src/main/resources/the_warlordResources/localization/eng/WarlordMod-Power-Strings.json

```diff
@@ -70,9 +75,9 @@
     ]
   },
   "the_warlord:ParryPower": {
-    "NAME": "",
+    "NAME": "Parrying Spirit",
     "DESCRIPTIONS": [
-      ""
+      "Whenever an attack removes all of your block, or you take no damage from an attack, parry. Parrying gives you random options from your #yParry #yDeck to play at the start of your next turn (After drawing cards). "
```

Comment:
The wording on this is very tricky to get right. I'd say something along the lines of "Whenever an attack causes you to lose all your block without causing you to lose HP, #yParry at the start of your turn. Choose and play 1 out of 3 random Parry cards in your Parry Deck.". Something like that. Or we just let Vertti think of a good way to phrase this ;)

Reply:
I think we should let Vertti handle this one, since you can have posture  and no block at all and still parry